droid-hal-device-img-boot.inc

```diff
@@ -98,6 +106,7 @@ Requires: oneshot
 %description
 %{summary}
 
+%if 0%{!?embed_recovery:1}
```

Comment:
As mentioned above, this %if never evaluates to true, because embed_recovery is now always defined (either to 1, or to 0). This also applies to all similar statements against `embed_recovery` below